src/main/java/frc/robot/RobotContainer.java

```diff
@@ -29,27 +38,61 @@
 public class RobotContainer {
     // The robot's subsystems
     private final Drivetrain m_robotDrive = new Drivetrain();
+    private final Elevator m_elevator;
+    private final Mechanism m_mechanism;
 
     // Flightstick controller
-    CommandJoystick m_driverFlightstickController = new CommandJoystick(OIConstants.kDriverControllerPort);
-    // XBox controller
-    CommandXboxController m_driverXboxController = new CommandXboxController(1);
+    CommandJoystick m_driverFlightstickController = new CommandJoystick(OIConstants.kFlightstickPort);
+    // Driving controller
+    CommandXboxController m_driverController = new CommandXboxController(OIConstants.kDrivingControllerXBoxPort);
+    // Operator controller
+    CommandXboxController m_operatorController = new CommandXboxController(OIConstants.kOperatingControllerXBoxPort);
 
     // The auto chooser
     private final SendableChooser<Command> autoChooser;
     // The input method chooser
     private final SendableChooser<Boolean> inputChooser;
 
+    private final boolean isRed = DriverStation.getAlliance().get() == DriverStation.Alliance.Red;
```

Comment:
I don't know if this will work on runtime. As at least at an event, the robot will not know the alliance until communication with FMS is connected.